recipients_app/lib/core/cubits/payment/payments_cubit.dart

```diff
@@ -49,9 +52,7 @@ class PaymentsCubit extends Cubit<PaymentsState> {
       emit(
```

Comment:
if there is an exception we will not have payments and we will not have paymentsUi. Also we want to notify crashlytics about that crash. It should be like this:

```
on Exception catch(ex, stackTrace) {
crashReportingRepository.reportCrash(...);
emit(state.copyWith(status: PaymentsStatus.failure, exception: ex));
}
```

Reply:
I did fix that already